runtime/jilgen/jilconsts.c

```diff
@@ -307,6 +307,9 @@ writeConstants(OMRPortLibrary *OMRPORTLIB, IDATA fd)
 #if defined(J9VM_JIT_32BIT_USES64BIT_REGISTERS)
 			writeConstant(OMRPORTLIB, fd, "ASM_J9VM_JIT_32BIT_USES64BIT_REGISTERS", 1) |
 #endif /* J9VM_JIT_32BIT_USES64BIT_REGISTERS */
+#if defined(J9VM_JIT_FREE_SYSTEM_STACK_POINTER)
+			writeConstant(OMRPORTLIB, fd, "ASM_J9VM_JIT_FREE_SYSTEM_STACK_POINTER", 1) |
+#endif /* J9VM_JIT_FREE_SYSTEM_STACK_POINTER */
```

Comment:
Just making sure, the `J9VM_JIT_FREE_SYSTEM_STACK_POINTER` define will still be defined on z/OS, but elsewhere and we will emit the `ASM_J9VM_JIT_FREE_SYSTEM_STACK_POINTER` M4 macro here so it gets activated in the M4 files rather than adding to the `M4_DEFINES`?

Reply:
J9VM_ will be defined for C/C++ only, ASM_J9VM_ for m4 assembly (or at least this is where we should end up).